README.md

```diff
@@ -40,6 +39,26 @@ if ($unleash->isFeatureDisabled('myAwesomeFeature')) {
 $allFeatures = $unleash->getFeatures();
 ```
 
+### Dynamic Arguments
+
+If your strategy relies on dynamic data at runtime, you can pass additional arguments to the feature check functions:
+
+```php
+use \MikeFrancis\LaravelUnleash\Unleash;
+use Config;
+
+$unleash = app(Unleash::class);
+
+$allow_list = config('app.allow_list');
```

Comment:
This is very pedantic but it would be great for consistency with the rest of the documentation/project if variables could be in camelCase:

```suggestion
$allowList = config('app.allowList');
```